### PR TITLE
doc/user: Revamp Third-Party Tools Page

### PR DESCRIPTION
**Pull request by ruf-io**
👉  Preview Link: https://preview.materialize.com/10962/third-party/supported-tools/ 👈 

### Motivation

Users will often come to us with questions about "Does Materialize integrate with X?"

- When the answer is **Yes**
   - Often the only way to know is to ask and get an answer in Slack or discussion. This page works as a self-serve index of tools and integrations that have already been looked at. 
   - The follow-up question of "how does it integrate" is not always well documented.
- When the answer is **no**
   - We don't have a great way of registering their interest, following up with them when the integration improves, and quantifying how many users are asking about certain integrations.

Users will also come to us asking "what tools can I use to do X with Materialize?" (e.g. manage SQL) - This page can also serve as a lightweight index for some of those questions.

### Review Criteria

- [x] The definitions of support levels has been reviewed.
- [x] All the claimed statuses of tools has been checked.
- [x] The "Notify Me" feature fires Segment events with user email, integration requested
- [x] Intro sentences for sections has been reviewed and approved.

### CODE REVIEW

File: doc/user/content/third-party/supported-tools.md [hunk 1]
```diff
@@ -1,33 +1,216 @@
...
+| Microsoft Power BI | 🔴 Researching | | [](#notify) |
+| Preset | 🔴 Researching | | [](#notify) |
+| Mode Analytics | 🔴 Researching | | [](#notify) |
+| Holistics BI | 🔴 Researching | | [](#notify) |
+
+### Other Data Applications and Tools
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Hex | 🟡 Alpha | Users of Hex can connect to Materialize Core instances via the Hex PostgreSQL connector. Hex automatically introspects Materialized Views and Tables. *(Cloud connectivity is blocked by user/password auth.)* | [](#notify) |
```
Comment (ruf-io) [inline doc/user/content/third-party/supported-tools.md:152]: @JLDLaughlin Can you suggest a status and note for Hex here

File: doc/user/content/third-party/supported-tools.md [hunk 2]
```diff
@@ -1,33 +1,216 @@
...
+## Data Tools
+
+Many tools in the modern data stack can connect to Materialize via PostgreSQL, but like ORMs, these tools sometimes make advanced configuration and system table queries to endpoints that Materialize hasn't yet implemented.
+
+### Business Intelligence (BI) Tools
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Metabase | 🟢 Beta | The Metabase PostgreSQL connector can be used to [connect Metabase to Materialize Core](/third-party/metabase/). | [More Info](/third-party/metabase/) [](#notify) |
+| Superset | 🟢 Beta | TBD @andrioni input. | [](#notify) |
```
Comment (ruf-io) [inline doc/user/content/third-party/supported-tools.md:139]: @andrioni Can you suggest a status and note for Superset here

File: doc/user/content/third-party/supported-tools.md [hunk 3]
```diff
@@ -1,33 +1,216 @@
...
+| Looker | 🟡 Alpha | TBD @andrioni input. | [](#notify) |
```
Comment (ruf-io) [inline doc/user/content/third-party/supported-tools.md:140]: @andrioni can you suggest a status and note for Looker here.

File: doc/user/content/third-party/supported-tools.md [hunk 4]
```diff
@@ -1,33 +1,216 @@
...
+| Google Data Studio | 🟡 Alpha | Google Data Studio can connect to Materialize Core and Cloud using the PostgreSQL connector. Data is cached hourly but can be manually refreshed. | [](#notify) |
+| Tableau | 🔴 Researching | | [](#notify) |
```
Comment (ruf-io) [inline doc/user/content/third-party/supported-tools.md:142]: @andrioni can you suggest a status and note for Tableau here.

File: doc/user/content/third-party/supported-tools.md (hunk 1 above)
Comment (JLDLaughlin) [reply doc/user/content/third-party/supported-tools.md:152]: Yes, sorry for my late response!

Status: alpha seems fine! Let's bump to beta once we've worked the partnership launch through with them. (And the known bugs.)
Note: it looks perfect to me! I'll open a PR when the user/password auth is fixed and when we get a custom connector logo.

**Comment by JLDLaughlin** [comment]
This page looks great, Andy!! Super excited about this.

File: doc/user/content/third-party/supported-tools.md (hunk 4 above)
Comment (andrioni) [reply doc/user/content/third-party/supported-tools.md:142]: Researching looks good! No other comments other than that, though.

File: doc/user/content/third-party/supported-tools.md (hunk 2 above)
Comment (andrioni) [reply doc/user/content/third-party/supported-tools.md:139]: IMO Alpha until DevEx is able to take a deeper dive, and then beta. We have no known remaining issues with Superset and everything should work. Superset should support the TLS certs that we use for Cloud, but I haven't tested it yet.

Comment (ruf-io) [reply doc/user/content/third-party/supported-tools.md:139]:
```suggestion
| Superset | 🟢 Alpha | Connect Superset to Materialize Core or Cloud using the Postgres connector.  | [](#notify) |
```

File: doc/user/content/third-party/supported-tools.md (hunk 3 above)
Comment (ruf-io) [reply doc/user/content/third-party/supported-tools.md:140]:
```suggestion
| Looker | 🟡 Alpha | Connect Looker to Materialize Core by adding a PostgreSQL 9.5+ database connection and specifying your Materialize credentials. Connections to Materialize Cloud are currently blocked by user/password auth. | [](#notify) |
```

**Review by bobbyiliev — APPROVED** [review]

**Pull request merged**